### DISCUSSION TITLE
Geoip location data retroactive?

### DISCUSSION THREAD

**Discussion by iwaffles** [plausible/analytics#738] [Self-Hosted Support] [answered]
I didn't have the geoip service running and once I noticed it wasn't running, I started it up. Will it find and add location data retroactively (for analytics that occurred while it wasn't running) or only for new requests? 

I'm assuming it won't, since the [IP address isn't stored anywhere](https://plausible.io/docs/countries), which may have answered this question :)

**Comment by Vigasaurus** [comment, marked as answer]
Your assumption is correct; the IP is used initially to determine the country, and then not stored, so retroactive application is unfortunately not possible.
